renovate.json

```diff
@@ -3,11 +3,22 @@
         "github>konflux-ci/mintmaker//config/renovate/renovate.json"
     ],
     "enabledManagers": [
```

Comment:
I wasn't aware of that, that would definitely be easier to manage.  I'm used to dependabot, where you have to specify everything.

Reply:
https://docs.renovatebot.com/modules/manager/#limiting-enabled-managers

Reply:
I was pulling in https://github.com/konflux-ci/mintmaker/blob/main/config/renovate/renovate.json, maybe I don't need to, since I see it's disabling some managers that I would find useful here (gomod in particular).

Reply:
Adjusted in b2a48e1.